src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -475,12 +481,11 @@ public void search(boolean onlyDataRelatedToMe) {
                     }
                 }
             }
-            
-
-            
+                        
             dataversePage.setQuery(query);
             dataversePage.setFacetCategoryList(facetCategoryList);
-            dataversePage.setFilterQueries(filterQueriesFinal);
+            dataversePage.setFilterQueries(filterQueriesDisplay);
```

Comment:
Is this setFilterQueries the crux of the bug? Of #6641? I see some conversation at https://github.com/IQSS/dataverse/pull/6473/commits/cbad0273542534fb41f083143d9016be7843a6cc#r360054910 about it was switched from `filterQueries` to `filterQueriesFinal` in cbad027 (part of pull request #6473). I guess I'm wondering if a one line change to switch it back from `filterQueriesFinal` to `filterQueries` would fix it the bug.

<img width="809" alt="Screen Shot 2020-08-04 at 12 21 56 PM" src="https://user-images.githubusercontent.com/21006/89319478-27198b00-d64e-11ea-9f89-d9278814065f.png">

Reply:
A one line fix, of course would be nice (and preferred) though we should make sure we understand why that line was changed in the first place. @sekmiller can you look at this and try to understand? Maybe build a branch with the one line change to see if it fixes it, then we can decide if how to get both issues to work?

Reply:
More than a one line fix but fairly simple. The addition of the DVobject types and subtree to the list of filter queries was necessary for the fix in #6268. I added a method to the SearchIncludeFragment getTypeFromFilterQuery in order to not render those filter queries in the search-include-fragment (see lines 341-342)

Reply:
I see. I kept wondering why we're adding more code in this pull request when I thought we'd be reverting something. Because the new types were added recently (DvObject and subtree), now we need to hide them, it sounds like.

Reply:
Right and also during development I noticed that the filter queries were being added multiple times hence the isFilterQueryAlreadyInMap method.